<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа #7</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>951</width>
      <height>731</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="but_clear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистить экран</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="but_lines">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ввести отрезки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_clip">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ввести отсекатель</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_add_lines">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить отрезки на
границах отсекателя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_clipping">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отсечь</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_centre_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удерживайте CTRL, чтобы
добавить горизонтальную линию</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lab_centre_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удерживайте SHIFT, чтобы
добавить вертикальную линию</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1269</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
